packages/core/execution-context/core-execution-context-browser-internal/BUILD.bazel

```diff
@@ -0,0 +1,110 @@
+load("@npm//@bazel/typescript:index.bzl", "ts_config")
+load("@build_bazel_rules_nodejs//:index.bzl", "js_library")
+load("//src/dev/bazel:index.bzl", "jsts_transpiler", "pkg_npm", "pkg_npm_types", "ts_project")
+
+PKG_DIRNAME = "core-execution-context-browser-internal"
+PKG_REQUIRE_NAME = "@kbn/core-execution-context-browser-internal"
+
+SOURCE_FILES = glob(
+  [
+    "src/**/*.ts",
+    "src/**/*.tsx",
+  ],
+  exclude = [
+    "**/*.test.*",
+  ],
+)
+
+SRCS = SOURCE_FILES
+
+filegroup(
+  name = "srcs",
+  srcs = SRCS,
+)
+
+NPM_MODULE_EXTRA_FILES = [
+  "package.json",
+]
+
+RUNTIME_DEPS = [
+  "@npm//lodash",
+  "@npm//rxjs",
```

Comment:
I wasn't sure how to handle imports from `rxjs/operators` in the bazel config file and changed all the operator imports to just use the main package.
Clearly, I didn't have to 😁 !

Reply:
Exactly, this part of the Bazel configuration is pretty similar to `package.json`. From what I've seen, here we can tell Bazel the different NPM (prefixed by `@npm/`) and local (prefixed by `/`) packages to be imported.